### PR TITLE
py-bba-data-push: Release 3.0.0

### PR DESCRIPTION
**Pull request by eleftherioszisis**

### CODE REVIEW

File: bluebrain/repo-bluebrain/packages/py-bba-data-push/package.py
Comment (lecriste) [inline bluebrain/repo-bluebrain/packages/py-bba-data-push/package.py:1]: v3.0.0 requires also
`depends_on("py-voxcell", type=("build", "run"))`

**Review by lecriste — DISMISSED** [review]
I wonder what [this](https://github.com/BlueBrain/spack/pull/2309#issuecomment-1943400694) means.

**Review by matz-e — CHANGES_REQUESTED** [review]
@eleftherioszisis needs a version restriction on `py-nexusforge`, with a new version added into that latter package.

**Review by lecriste — DISMISSED** [review]
@matz-e, by adding 
`depends_on("py-nexusforge@0.8.1", type=("build", "run"), when="@3.0.0:")`
what happens to the `py-nexusforge` version of v1.0.8 if we don't add
`depends_on("py-nexusforge@0.8.0", type=("build", "run"), when="@1.0.8")`
?

**Comment by eleftherioszisis** [comment]
I updated py-bbp-workflow and py-cwl-registry to play nicely with py-bba-data-push 3.0.0.

**Review by matz-e — APPROVED** [review]

**Comment by matz-e** [comment]
> @matz-e, by adding `depends_on("py-nexusforge@0.8.1", type=("build", "run"), when="@3.0.0:")` what happens to the `py-nexusforge` version of v1.0.8 if we don't add `depends_on("py-nexusforge@0.8.0", type=("build", "run"), when="@1.0.8")` ?

If you don't add the second line (or a generic dependency on `py-nexusforge`), Spack will not see any dependency on nexus forge at all.

**Pull request merged**